sycl/test-e2e/ESIMD/lsc/Inputs/lsc_slm_load.hpp

```diff
@@ -146,7 +146,7 @@ bool test(queue Q, uint32_t PMask = ~0) {
           Tuint ExpectedVal =
               IsMaskSet ? (ExpectedValBase + ChannelId + J * NChannels) & VMask
                         : sycl::bit_cast<Tuint>(MergeValue);
-          Tuint ComputedVal = sycl::bit_cast<Tuint>(Out[OutIndex]);
+          Tuint ComputedVal = sycl::bit_cast<Tuint>(Out[OutIndex]) & VMask;

```

Comment:
Is this related to what we discussed with Victor recently?

Reply:
I don't remember which test was shown to Victor.
This test failed on PVC with opencl and I fixed the issue.

Reply:
If this one is different case, then it is similar.

Reply:
Thank you.
BTW, @turinevgeny says is the same test that we looked at yesterday.